kolibri/core/auth/management/commands/bulkexportusers.py

```diff
@@ -39,16 +40,34 @@
 # TODO: decide whether these should be internationalized
 labels = OrderedDict(
     (
-        ("id", _("Database ID ({})".format("UUID"))),
-        ("username", _("Username ({})".format("USERNAME"))),
-        ("password", _("Password ({})".format("PASSWORD"))),
-        ("full_name", _("Full name ({})".format("FULL_NAME"))),
-        ("kind", _("User type ({})").format("USER_TYPE")),
-        ("id_number", _("Identifier ({})".format("IDENTIFIER"))),
-        ("birth_year", _("Birth year ({})".format("BIRTH_YEAR"))),
-        ("gender", _("Gender ({})".format("GENDER"))),
-        ("enrolled", _("Learner enrollment ({})".format("ENROLLED_IN"))),
-        ("assigned", _("Coach assignment ({})".format("ASSIGNED_TO"))),
+        ("id", _("Database ID ({})").format("UUID")),
+        ("username", _("Username ({})").format("USERNAME")),
+        ("password", _("Password ({})").format("PASSWORD")),
+        ("full_name", _("Full name ({})").format("FULL_NAME")),
+        (
+            "kind",
+            pgettext_lazy(
+                "CSV header for the type of user: ADMIN, LEARNER, COACH...",
+                "User type ({})",
+            ).format("USER_TYPE"),
+        ),
+        ("id_number", _("Identifier ({})").format("IDENTIFIER")),
+        ("birth_year", _("Birth year ({})").format("BIRTH_YEAR")),
+        ("gender", _("Gender ({})").format("GENDER")),
+        (
+            "enrolled",
+            pgettext_lazy(
+                "CSV file header for the list of classrooms names where the learner is going to be enrolled",
```

Comment:
As above `CSV column header for...`

Reply:
Change all the instances of `CSV file header...` below for `CSV column header...`.

Reply:
ok, I am changing this, even if I am not sure it's important or even correct, but honestly, I don't care too much about this wording as it's not user face.